src/CreditCardForm.js

```diff
@@ -163,16 +163,31 @@ export default class CreditCardForm extends Component {
       loading: true,
     }
     this.form = {};
+    this.loadingTimeOut = null
   }
 
   componentDidMount() {
-    const { apiOptions = {} } = this.props
+    const { apiOptions = {}, loadingState } = this.props
     configureVault(
       apiOptions,
       this.initialize,
     )
+
+    // setup debug information when using `loadingState`
+    if(!!loadingState === true && this.isThisMethod()){
```

Comment:
Why is form timeout only reported when using `loadingState`?  It could be a useful error message even when not using `loadingState`.